src/olympia/users/models.py

```diff
@@ -87,25 +87,30 @@ def widget_attrs(self, widget):
 
 class UserManager(BaseUserManager, ManagerBase):
 
-    def create_user(self, username, email, fxa_id=None):
+    def create_user(self, username, email, fxa_id=None, **kwargs):
         # We'll send username=None when registering through FxA to generate
         # an anonymous username.
         now = timezone.now()
         user = self.model(
-            username=username, email=email, fxa_id=fxa_id,
-            last_login=now)
+            username=username,
+            email=email,
+            fxa_id=fxa_id,
+            last_login=now,
+            **kwargs
+        )
         if username is None:
             user.anonymize_username()
         log.debug('Creating user with email {} and username {}'.format(
             email, username))
         user.save(using=self._db)
         return user
 
-    def create_superuser(self, username, email, fxa_id=None):
+    def create_superuser(
+            self, username, email, fxa_id=None):
```

Comment:
I guess line break not needed anymore?

Reply:
No, I don't know why there was a break.

Reply:
Can we get rid of this line-break before merging please?